modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/segment/pages/ProfileRoutes.jsx

```diff
@@ -197,6 +200,25 @@ export class SegmentProfileRoutes extends React.Component {
 					/>
 				</BasePage.Header>
 
+				{getMatchedRoute(NAV_ITEMS) === Routes.CONTACTS_SEGMENT && (
+					<BasePage.SubHeader>
+						<div className='d-flex justify-content-end w-100'>
+							<DownloadPDFReport
+								containers={[
+									Containers.SegmentMembershipCard,
+									Containers.SegmentCompositionCard,
+									Containers.SegmentCriteriaCard,
+									Containers.TopInterests,
+									Containers.DistributionBreakdownCard
+								]}
+								disabled={false}
```

Comment:
The same thing here.

Reply:
@interaminense I can't destructure `const dataSourceStates = useDataSource();` here because ProfileRoutes.jsx is a class component. Can I keep passing "false" on disabled here?

Reply:
We need to check data source yet, even it is a class component.

Reply:
Note: We decided to find a solution for this later because we need to deliver this ASAP